pkg/serverless/tags/tags.go

```diff
@@ -229,9 +230,18 @@ func getRuntimeFromOsReleaseFile(osReleasePath string) string {
 	return runtime
 }
 
-func getRuntime(procPath string, osReleasePath string, varName string) string {
-	foundRuntimes := proc.SearchProcsForEnvVariable(procPath, varName)
-	runtime := cleanRuntimes(foundRuntimes)
+func getRuntime(procPath string, osReleasePath string, varName string, retries int) string {
```

Comment:
LGTM, I would only add a small comment to explain why we're retrying as we're likely to forget it later